Xamarin.PropertyEditing.Mac/Themes/MacThemeManager.cs

```diff
@@ -1,12 +1,14 @@
 ﻿using System;
 using System.Drawing;
+using System.Collections.Generic;
 using AppKit;
 
 namespace Xamarin.PropertyEditing.Themes
 {
 	public class MacThemeManager : BaseThemeManager
 	{
 		public NSAppearance CurrentAppearance { get; private set; }
+		private Dictionary<string, NSImage> themeCache = new Dictionary<string, NSImage> ();
```

Comment:
Might be worth clearing this when the theme changes.

Reply:
Possibly. If the user changes between themes we'd be taking the hit again on adding the items to the new list. Without the clear, we'd just keep adding and right now we only have 2 possible themes. So do we want to take that hit or can we live with the extra memory it would take up, holding the images in memory for 2 themes?

Reply:
The hit is relatively minor, we're not even caching currently. In practice a user isn't going to be switching themes constantly, it's not worth wasting a bunch of memory over.